deployment/terraform/modules/rds/main.tf

```diff
@@ -1,5 +1,5 @@
 resource "aws_db_subnet_group" "db_subnet_group" {
-  name       = "rds subnet group"
+  name       = "${var.name}"
```

Comment:
Why is name necessary?

Reply:
Without this, I was getting an error about the name already being in use. I guess it should have recognized that the resource it needed already existed, but it didn't work that way for me... perhaps there was some other aspect of my environment which was weird, but if you're pretty confident this is unnecessary, we can revert.

Reply:
I have never had this error. I suspect there was something wrong in your environment.